### PR TITLE
r-cran-*: updates and fixes

### PR DESCRIPTION
**Pull request by q66**
Some of these neeeded updating as archive didn't have the releases anymore.

### CODE REVIEW

File: srcpkgs/R-cran-pkgconfig/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'R-cran-pkgconfig'
+pkgname=R-cran-pkgconfig
+version=2.0.2
+revision=1
+build_style=R-cran
+short_desc="Private Configuration for 'R' Packages"
+maintainer="Florian Wagner <florian@wagner-flo.net>"
```
Comment (maxice8) [inline srcpkgs/R-cran-pkgconfig/template]: Please us your own name when introducing new packages

**Comment by q66** [comment]
I will update things once I'm back from FOSDEM.

**Comment by q66** [comment]
fixed

**Comment by maxice8** [comment]
Welcome back, hope FOSDEM was fun

**Comment by q66** [comment]
Yeah, FOSDEM was great. The amount of stuff to rebuild after returning is not so great, but oh well

**Pull request merged**